### DISCUSSION TITLE
Bootable from Stick but not from HardDisk drive ...need help for solution

### DISCUSSION THREAD

**Discussion by SashonIT** [osy86/HaC-Mini#1064] [Q&A]
I created a OpenCore 090 Install Stick.

I installed MacOS 13.0 Ventura.

In MacOS Ventura 13.0 bootet from the Stick i installed the  HacMini 3.2.2 Installer.

With Thunderbolt NAtive
Whatever Green
Lilu
And DW 1820 Bluetooth/Network

Insatll was Succesfull.

Then i updated the EFI to Open Core 090.

Copied in the EFI Folder of the HDD.

When i boot now from HDD the Boot Mechanism stops at ( You can See in the Picture attached)

![image](https://user-images.githubusercontent.com/126977959/225275334-92a5661d-dc8b-44c6-b4f9-af09d5fb391d.png)

Why is it Booting from Stick flawlessly but not from HDD?

**Comment by SashonIT** [comment]
Er lebt!!!

Des Rätsels Lösung war folgendes!!!!

Habe einfach mal Hängenbleibt Zeile inse Google getan....
[PCI configuration end, bridges 14 devices 24]

Dann kam ein Reddit Link mit selben Problem nur anderen Zahlen heraus.

Darin Versteckt war folgender Hinweis:



Habe ich erst nicht verstanden! Erneutes Guugeln ---> brachte mir Folgendes Erjebnis:



Seit dem läuft es endlich OHNE den [Install Stick](https://www.hackintosh-forum.de/lexicon/entry/122-install-stick/?synonym=234) und direkt von Feschtplatte!

Alles Funktioniert soweit......

Nur leider Ohne Thunderbolt 3, was ich dringend benötige.

Kann mir da vielleicht jemand hier unterstützen? Oder mehrere?

Achja.... Airdrop Empfang von iPhone und oder iPad ist möglich.

Nur leider kein Versand von dieser Maschine.

Hat das jemand von euch schonmal gehabt dieses Einbahnstrassen Airdrop?

**Comment by ivan-ef** [comment]
you should search this repository for a solution for all your issues. https://github.com/osy/HaC-Mini/issues
